app/static/scss/admin/users/users.scss

```diff
@@ -14,6 +14,73 @@
         margin-top: 1rem;
         display: block;
     }
+    @media screen and (min-width: 641px) {
+        .display-feature-table {
+            table, td, th {
+                overflow: hidden;
+                border: 1px solid #e0e0e0;
+            }
+            .highlighted {
```

Comment:
There's still a small range where the table looks like this, but it looks better now. 
Making the width any larger would cause the "x" sign to go off-screen.
![screen shot 2015-04-24 at 5 05 37 pm](https://cloud.githubusercontent.com/assets/7966567/7328528/49cdec0e-eaa4-11e4-9073-d8b7a048cb17.png)